modules/microshift-storage-backup-volume-snapshots.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// microshift/volume-snapshots-microshift.adoc
+
+:_content-type: PROCEDURE
+[id="backing-up-a-volume-snapshotting_{context}"]
+= Backing up a volume snapshot
+
+Snapshots of {product-title} data are created as read-only logical volumes (LVs) located on the same devices as the original data. All local volumes must be manually mounted before they can be copied as persistent volumes (PVs) and used as backup copies. To use a snapshot of a {product-title} storage volume as a backup, you can locate it on the local host and then move it to a secure location. To find specific snapshots and copy them, use the following procedure.
+//are these LVs or PVs?
+
+.Prerequisites
+
+. You have root access to the host machine.
+. A {product-title} snapshot was successfully created.
+
+.Procedure
+
+. Get the `volumeSnapshot.status.volumeSnapshotContentName` by using the following command syntax:
++
+[source, terminal]
+----
+$ oc get volumesnapshot -n test-ns my-snap -o yaml
```

Comment:
This is going to dump a lot more data than the single value they need.

I think we want

```suggestion
$ oc get volumesnapshot -n test-ns my-snap -o 'jsonpath={.status.volumeSnapshotContentName}'
```

@copejon can you confirm I have that path right, please?